### ISSUE TITLE
Add binding for helm-gtags-find-tag-other-window

### ISSUE THREAD

**Issue by dubnde** [syl20bnr/spacemacs#5742]
#### Description
Add `SPC m g D` binding for `helm-gtags-find-tag-other-window`. This will mirror `SPC m g d` for `helm-gtags-find-tag`

#### Reproduction guide
- Start Emacs
- Open major mode file like a C/C++ file
- try to find tag using `SPC m g`

We can see the binding for `helm-gtags-find-tag` but none for `helm-gtags-find-tag-other-window`

*Observed behaviour:*
No binding for `helm-gtags-find-tag-other-window`

*Expected behaviour:*
Binding for `helm-gtags-find-tag-other-window`

#### System Info
- OS: darwin
- Emacs: 24.5.1
- Spacemacs: 0.105.17
- Spacemacs branch: develop (rev. 5c7e0b6)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
((auto-completion :variables auto-completion-tab-key-behavior 'cycle :disabled-for shell)
 better-defaults
 (c-c   :variables c-c  -enable-clang-support t c-c  -default-mode-for-headers 'c  -mode)
 emacs-lisp evil-cleverparens evil-commentary
 (evil-snipe :variables evil-snipe-enable-alternate-f-and-t-behaviors t)
 git gtags javascript markdown org osx semantic
 (shell :variables shell-enable-smart-eshell t)
 shell-scripts spacemacs-helm spell-checking syntax-checking version-control vim-powerline windows-scripts yaml)
```

**Comment by TheBB** [comment]
Thanks, but we don't need reproduction steps for a feature request. :-)

**Comment by bmag** [comment]
PR pending: #5587

**Comment by TheBB** [comment]
Doesn't `helm-gtags-find-tag` already have an action for opening in another window?

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
